tests/integration/test_integration.py

```diff
@@ -0,0 +1,96 @@
+import time
+import pytest
+import rethinkdb as r
+
+from bigchaindb import Bigchain
+
+
+@pytest.fixture
+def inputs(user_vk):
+    from bigchaindb.models import Transaction
+
+    b = Bigchain()
+
+    # create blocks with transactions for `USER` to spend
+    for block in range(4):
+        transactions = [
+            Transaction.create(
+                [b.me], [user_vk], metadata={'i': i}).sign([b.me_private])
+            for i in range(10)
+        ]
+        block = b.create_block(transactions)
+        b.write_block(block, durability='hard')
+
+
+@pytest.mark.usefixtures('processes')
+def test_fast_double_create(b, user_vk):
```

Comment:
Moved this to https://github.com/bigchaindb/bigchaindb/issues/832 so this can be merged without it.